girder/utility/server.py

```diff
@@ -248,7 +250,7 @@ def setup(test=False, plugins=None, curConfig=None):
     cherrypy.tree.mount(girderWebroot.api, '/api', appconf)
 
     # Mount everything else in the routeTable
-    for (name, route) in six.viewitems(routeTable):
+    for name, route in six.viewitems(routeTable):
```

Comment:
Is this actually a failing linting test?

Reply:
It wasn't, I am just not a fan of unnecessary parens unless they add some readability.